src/frontend/src/lib/components/ui/IdentitySwitcher.svelte

```diff
@@ -112,6 +115,10 @@
     </FeaturedIcon>
     <span>Use another identity</span>
   </ButtonCard>
+  {#if onLogout}
+    <Button onclick={onLogout} variant="tertiary"><LogOutIcon />Sign Out</Button
```

Comment:
Nitpick: icon size in design is 1rem for default button size if I remember correctly.

```suggestion
    <Button onclick={onLogout} variant="tertiary"><LogOutIcon size="1rem" />Sign Out</Button
```